packages/global-styles/src/components/global-styles-variations/index.tsx

```diff
@@ -105,6 +107,26 @@ const GlobalStylesVariations = ( {
 	splitPremiumVariations = true,
 	displayFreeLabel = true,
 }: GlobalStylesVariationsProps ) => {
+	const [ , experiment ] = useExperiment( 'calypso_global_styles_personal' );
+	const globalStylesOnPersonalExperiment = experiment?.variationName === 'treatment';
+	const [ premiumStylesDescription, setPremiumStylesDescription ] = useState( '' );
+	useEffect( () => {
+		if ( globalStylesOnPersonalExperiment ) {
+			setPremiumStylesDescription(
+				translate(
+					'Unlock custom styles and tons of other features with the Personal plan, or try them out now for free.'
+				)
+			);
+			return;
+		}
+
+		setPremiumStylesDescription(
+			translate(
+				'Unlock custom styles and tons of other features with the Premium plan, or try them out now for free.'
+			)
+		);
+	}, [ globalStylesOnPersonalExperiment ] );
```

Comment:
This looks quite overkill. Cannot we do just this?

```suggestion
	const premiumStylesDescription = globalStylesOnPersonalExperiment
		? translate(
					'Unlock custom styles and tons of other features with the Personal plan, or try them out now for free.'
				)
		: translate(
				'Unlock custom styles and tons of other features with the Premium plan, or try them out now for free.'
			);
```

Reply:
With the code you're suggesting we would be translating every time the component gets rendered, even though the translation is going to be the same.